include/nbl/system/ISystem.h

```diff
@@ -112,14 +112,15 @@ class NBL_API2 ISystem : public core::IReferenceCounted
         };
 
         //! Compile time resource ID
-        template<typename StringUniqueType>
+        template<nbl::core::StringLiteral Path>
         inline core::smart_refctd_ptr<const IFile> loadBuiltinData() const
         {
         #ifdef _NBL_EMBED_BUILTIN_RESOURCES_
-            return impl_loadEmbeddedBuiltinData(StringUniqueType::value,nbl::builtin::get_resource<StringUniqueType>());
+
+            return impl_loadEmbeddedBuiltinData(Path.value, nbl::builtin::get_resource<Path>());
         #else
             future_t<core::smart_refctd_ptr<IFile>> future;
-            createFile(future,StringUniqueType::value,core::bitflag(IFileBase::ECF_READ)|IFileBase::ECF_MAPPABLE);
+            createFile(future, STRING_LITERAL::value,core::bitflag(IFileBase::ECF_READ)|IFileBase::ECF_MAPPABLE);
```

Comment:
this can't work

Reply:
em I believe that was your change I didn't look at XD

Reply:
fixed it myself